.shippable.yml

```diff
@@ -82,18 +90,10 @@ build:
             cp ./scripts/sanity_chk/last_sanity.xml shippable/testresults/;
             aws s3 cp ./scripts/sanity_chk/last_sanity.xml ${S3_PATH}/sanitycheck.xml;
           fi;
-    on_failure:
```

Comment:
yeah, both on_success and on_failure had exactly the same content, so there was duplication.